### PR TITLE
gh-2335 Adding github workflow to automate koryphe version update.

### PR DESCRIPTION
**Pull request by m29827**

### CODE REVIEW

**Comment by m29827** [comment]
@d47853 This change is still using the updateKorypheVersion.sh script to perform the update. Happy to migrate the logic into the workflow but it might be useful to keep the script intact so it can be run outside of github.

File: .github/workflows/update-koryphe-version.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        run: |
+          git config user.name Gaffer
+          git config user.email github-actions@github.com
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Update Korpyhe Version
```
Comment (d47853) [inline .github/workflows/update-koryphe-version.yaml:32]: Could make the pull request afterwards?

**Review by d47853 — COMMENTED** [review]
Looks good. Although it might be worth automatically submitting a pull request if the version is upgraded.

File: .github/workflows/update-koryphe-version.yaml (hunk 1 above)
Comment (m29827) [reply .github/workflows/update-koryphe-version.yaml:33]: Now creating a PR.

File: .github/workflows/update-koryphe-version.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        run: ./cd/updateKorypheVersion.sh ${{ github.event.inputs.version }}
+
+      - name: Submit PR
+        if: ${{ success() }}
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: ${{ env.VERSION_UPDATE_BRANCH }}
+          destination_branch: develop
+          github_token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```
Comment (d47853) [inline .github/workflows/update-koryphe-version.yaml:42]: Don't think admin token would be required for this. You could use the default one:
```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

**Review by d47853 — CHANGES_REQUESTED** [review]

**Review by d47853 — APPROVED** [review]

**Pull request merged**